File: Api/ViewModels/TokenEntity.cs
namespace PublishPlatform.Api.ViewModels
{
    /// <summary>
    /// Token Entity
    /// </summary>
    public class TokenEntity
    {
        /// <summary>
        /// Token
        /// </summary>
        public string Token { get; set; }

        /// <summary>
        /// ExpiresIn
        /// </summary>
        public int ExpiresIn { get; set; }
    }
}

File: Client/src/models/TokenEntity.ts
export class TokenEntity{
    
    private _token : string;
    public get token() : string {
        return this._token;
    }
    public set token(v : string) {
        this._token = v;
    }
    
    private _expiresIn : number;
    public get expiresIn() : number {
        return this._expiresIn;
    }
    public set expiresIn(v : number) {
        this._expiresIn = v;
    }
}